### DISCUSSION TITLE
Question about new documentation page

### DISCUSSION THREAD

**Discussion by Markmaster** [blinksh/blink#1973] [Support] [answered]
Hi,

I have a question about [this](https://docs.blink.sh/integrations/tailscale+mosh) page and need clarification.
It says in Step 1:
`Open the Blink app on your device, and in the Blink Shell run the command ssh username@<hostname> where username is your Tailscale user name, and hostname is your Tailscale Hostname or IP address.`
If I log in to the TailScale via my Gmail account, what is my username for this? username@gmail.com@hostname? For me double @ doesn't make sense. 
And the same for Step 2:
`Now, it is time to upgrade our connection to Mosh. Run the command mosh --install-static <username>@hostname where username and hostname are the same parameters you used before. Blink will automatically request permission to install mosh-server on your host, and set up everything for you.`
Also, I've got an error while trying to install it:
`blink> mosh --install-static --verbose username@hostname
socket_callback_connected: Socket connection callback: 1 (0)
ssh_known_hosts_read_entries: Failed to open the known_hosts file '/etc/ssh/ssh_known_hosts': No such file or directory
bootstrapMoshServer: Trying bootstrap with sequence: [Blink.UseMoshOnPath, Blink.InstallStaticMosh], experimental: BKMoshExperimentalIP(rawValue: 0), family: nil, args: new -s -c 256 -l LANG=en_US.UTF-8
bootstrapMoshServer: Trying Blink.UseMoshOnPath
bootstrapMoshServer: Connecting to ~/.local/blink/mosh-server new -s -c 256 -l LANG=en_US.UTF-8
bootstrapMoshServer: Command output: /bin/bash: line 1: /home/markmaster/.local/blink/mosh-server: No such file or directory

bootstrapMoshServer: Bootstrap failed with NoMoshServerArgs
bootstrapMoshServer: Trying Blink.InstallStaticMosh
platformAndArchitecture: uname output: ["Linux", "x86_64", ""]
getMoshServerBinary: linux x86_64
getMoshServerBinary: Downloading https://github.com/blinksh/mosh-static-multiarch/releases/latest/download/mosh-server-1.4.0-linux-x86_64
getMoshServerBinary: Download mismatch. Downloaded size: 9
NoChecksumMatch - Error downloading binary. The checksums do not match.
Use mosh1 for the deprecated (previous) mosh version.
blink> `

**Comment by iflowerml** [comment, marked as answer]
Hi, thanks for reaching out!
Apologies for the confusion. You are correct we are not referring to the Tailscale username. We are referring to the username in the remote machine you want to log into. We will make the proper corrections in the article so this is clearer.

For Step 1, the Tailscale username refers to the username to log into the target machine. In your terminal it should look like `root@ubuntu-123`. `root` would be the username. For IP address you would use the Tailscale tailnet IP you get from `tailscale ip`.

For Step 2, likewise use the username of the target, and the Tailscale IP of the machine. What you describe to bootstrap mosh-server is a known issue and is being tracked (looks like you already posted there too, so stay tuned) https://github.com/blinksh/blink/issues/1963.

**Reply by Markmaster** [reply]
Thank you for the clarification.
